src/main/java/setup/cardlists/MakeAllCardsList.java

```diff
@@ -15,37 +15,46 @@
 public class MakeAllCardsList {
 
 	//This code is meant to be run using some kind of scheduler.
+	private static boolean test = false;
 
-	public static void main(String[] args) throws Exception{
+	public static void main(String[] args) throws Exception {
 
-		final int TIMES_CHECKED = 168; //Hours in a week
-		//final int TIMES_CHECKED = 1; //For testing
+		final int TIMES_CHECKED = (test ? 1 : 168); // 168 hours per week
 
 		//Create a list of set abbreviations from the provided file, and check how many times the program has been run so far.
-		List<String> setAbbr; 
+		List<String> setAbbr;
 
-		if (!new File("MTGO_Set_Abbreviations").exists())
-		{
-			setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations"));
-		}
-		else {
-			setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations");			
+		if (test) {
+			if (!new File("MTGO_Set_Abbreviations_test").exists())
+			{
+				setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations_test"));
+			}
+			else {
+				setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations_test");
+			}
+		} else {
+			if (!new File("MTGO_Set_Abbreviations").exists())
+			{
+				setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations"));
+			}
+			else {
+				setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations");
+			}
 		}
-		if (!new File("Count.txt").exists())
-		{
+
+		if (!new File("Count.txt").exists()) {
 			FileConverter.writeFile(0, "Count.txt");
 		}
 		int count = FileConverter.readToInt("Count.txt");
 
 		//Increase the count of how many times this program has been run, and update the text file to reflect that.
 		count++;
 
-		if (count < TIMES_CHECKED){
```

Comment:
Nit. This change makes sense but ideally would be in a separate commit for ease of review.